src/utils/case-studies-data.js

```diff
@@ -957,6 +956,14 @@ export default {
       },
     ],
   },
+  tanzu: {
+    iconName: 'tanzu',
```

Comment:
this also appear not in alphabetical order on the page

Reply:
Ah yes, figured it out, I used tanzu for the card name instead of VMware Tanzu